umh-core/pkg/diagnosis/engine.go

```
@@ -0,0 +1,436 @@
+		tracks:  make([]trackState[S], 0, len(t.Tracks)),
+	}
+	for _, s := range t.Signals {
+		for _, inst := range s.Instruments {
+			w, err := NewSlidingWindow(inst.Span, s.DemoteSpan, inst.Red, inst.Counter)
+			if err != nil {
+				return nil, err
+			}
+			e.windows[key{Signal: s.Name, Instrument: inst.Name}] = w
+		}
+	}
+	for _, tr := range t.Tracks {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above range)

(wsl_v5)